config/sys.config

```diff
@@ -27,6 +27,7 @@
   ]},
 
   {aecore, [
+      {beneficiary, <<"ak$tjnw1KcmnwfqXvhtGa9GRjanbHM3t6PmEWEWtNMM3ouvNKRu5">>},
```

Comment:
I think that adding default public key is no go, as if users don't configure it they'll loose their rewards